Tools/AP_Bootloader/board_types.txt

```diff
@@ -404,6 +404,9 @@ AP_HW_CUAV-7-NANO                    7000
 # IDs 7100-7109 reserved for VIEWPRO
 AP_HW_VUAV-V7pro                     7100
 
+# IDs 7200-7209 reserved for AEROFOX
+AP_HW_AEROFOX_H7                     7200
```

Comment:
This needs to be in a separate commit

Reply:
Ditto the bootloader

Reply:
I'll submit to modify it in the next commit